<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>udg::QBasicSearchWidgetBase</class>
 <widget class="QWidget" name="udg::QBasicSearchWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Basic search</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Patient</string>
       </property>
       <layout class="QGridLayout">
        <property name="margin">
         <number>9</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="1">
         <widget class="QLineEdit" name="m_patientNameText">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="textLabel2_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Patient Name</string>
          </property>
          <property name="buddy">
           <cstring>m_patientNameText</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="textLabel1_3">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Patient ID</string>
          </property>
          <property name="buddy">
           <cstring>m_patientIDText</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLineEdit" name="m_patientIDText">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Study Date</string>
         </property>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>9</number>
          </property>
          <item>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="m_anyDateRadioButton">
              <property name="text">
               <string>Any date</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_todayRadioButton">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Today</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_yesterdayRadioButton">
              <property name="text">
               <string>Yesterday</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_lastWeekRadioButton">
              <property name="text">
               <string>Last Week</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>186</width>
                <height>23</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QRadioButton" name="m_customDateRadioButton">
              <property name="text">
               <string>Custom:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <spacer>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Maximum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>30</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QVBoxLayout">
                <property name="spacing">
                 <number>6</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <layout class="QHBoxLayout">
                  <property name="spacing">
                   <number>6</number>
                  </property>
                  <property name="margin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QCheckBox" name="m_fromDateCheck">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>From</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDateEdit" name="m_fromStudyDate">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="wrapping">
                     <bool>false</bool>
                    </property>
                    <property name="buttonSymbols">
                     <enum>QAbstractSpinBox::UpDownArrows</enum>
                    </property>
                    <property name="calendarPopup">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout">
                  <property name="spacing">
                   <number>6</number>
                  </property>
                  <property name="margin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QCheckBox" name="m_toDateCheck">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>To</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDateEdit" name="m_toStudyDate">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="calendarPopup">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="udg::QModalitiesSelectorGroupBox" name="m_buttonGroupModality">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Series Modality</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>udg::QModalitiesSelectorGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qmodalitiesselectorgroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_fromDateCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_fromStudyDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_toDateCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_toStudyDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_customDateRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_fromDateCheck</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_customDateRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_fromDateCheck</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_customDateRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_toDateCheck</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_customDateRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_toDateCheck</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
